src/documentation/pages/Scoping/index.html

```diff
@@ -0,0 +1,124 @@
+@@include('../../templates/modules/components/DocumentationPageHeader.html')
```

Comment:
These demo files will probably need to be cleaned up/refactored a bit, since I'd rather not provide a separate file to "mock" potential conflicts.